clients/privacy-center/cypress/e2e/consent-i18n.cy.ts

```diff
@@ -902,6 +908,259 @@ describe("Consent i18n", () => {
         });
       });
     });
+    describe("experience translation overrides", () => {
+      describe("when set via window obj", () => {
+        describe("when fides_override_language exactly matches experience locale", () => {
+          beforeEach(() => {
+            visitDemoWithI18n({
+              navigatorLanguage: ENGLISH_LOCALE,
+              globalPrivacyControl: true,
+              fixture: "experience_banner_modal.json",
+            });
```

Comment:
Does this... work? When you call `stubConfig` again later I think you will end up just blowing away the results of this `beforeEach`?

Reply:
🤔

Reply:
Wow, not sure how this was working, but I've fixed it!